test/e2e/config/provider.yaml

```diff
@@ -79,7 +79,7 @@ providers:
 variables:
   # default variables for the e2e test; those values could be overridden via env variables, thus
   # allowing the same e2e config file to be re-used in different prow jobs e.g. each one with a K8s version permutation
-  KUBERNETES_VERSION: "v1.22.0"
+  KUBERNETES_VERSION: "v1.22.3"
   ETCD_VERSION_UPGRADE_TO: "3.4.9-0"
   COREDNS_VERSION_UPGRADE_TO: "1.7.0"
   KUBERNETES_VERSION_UPGRADE_TO: "v1.22.0"
```

Comment:
I think we need to update the `KUBERNETES_VERSION_UPGRADE_TO` variable also.

Reply:
we don't have any upgarde tests. That is an unused variable. So left it as is.